### ISSUE TITLE
Localstack fails on creation of IAM ManagedPolicy through CloudFormation

### ISSUE THREAD

**Issue by Blikingor** [localstack/localstack#3513]
# Type of request: This is a ...

[x ] bug report
[ ] feature request

# Detailed description
When creating a CloudFormation stack containing a IAM ManagedPolicy, localstack fails with the message: `Unable to extract attribute "PhysicalResourceId" from "IAM::ManagedPolicy"`

The error only occurs if a LOCALSTACK_API_KEY is used.

## Expected behavior
The stack and policy should create successfully.

## Actual behavior
Error from AWS CLI:
```
$ aws cloudformation create-stack --stack-name test1 --template-body file://iam-managed-policy.json --endpoint=http://localhost:4566 --region eu-west-1

Unable to parse response (not well-formed (invalid token): line 4, column 143), invalid XML received. Further retries may succeed:
<ErrorResponse xmlns="http://cloudformation.amazonaws.com/doc/2010-05-15/"><Error>
        <Type>Sender</Type>
        <Code>ValidationError</Code>
        <Message>Unable to create stack "test1": Unable to extract attribute "PhysicalResourceId" from "IAM::ManagedPolicy" model class <class 'localstack.services.cloudformation.service_models.GenericBaseModel'></Message>
        </Error><RequestId>7dbaa525</RequestId>
        </ErrorResponse>
```

# Steps to reproduce

## Command used to start LocalStack

docker-compose:
```
version: '3'
services:
  localstack:
    image: localstack/localstack:latest
    ports:
      - "4565-4584:4565-4584"
      - "443:4566"
      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
    environment:
      #- SERVICES=sqs,lambda,s3,dynamodb,cloudformation,sts,apigateway,iam
      - DOCKER_HOST=unix:///var/run/docker.sock
      - HOSTNAME=localstack
      - HOSTNAME_EXTERNAL=${IPADDR}
      - DEFAULT_REGION=eu-west-1
      - DEBUG=1
      - DATA_DIR=${DATA_DIR- }
      - PORT_WEB_UI=${PORT_WEB_UI- }
      - LAMBDA_EXECUTOR=docker
      - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
      - LOCALSTACK_API_KEY=myvalidkey
      - https_proxy=http://myprox:8080
      - http_proxy=http://myprox:8080
    volumes:
      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

- AWS CLI:

```
$ aws cloudformation create-stack --stack-name test1 --template-body file://iam-managed-policy.json --endpoint=http://localhost:4566 --region eu-west-1
```

- CloudFormation template `iam-managed-policy.json`:

```
{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Description": "",
  "Resources": {
    "AccessLogs": {
      "Type": "AWS::Logs::LogGroup",
      "Properties": {
        "LogGroupName": "/org/bps/d/ew1/abcde/client"
      }
    },
    "LogGroupPolicy": {
      "Type": "AWS::IAM::ManagedPolicy",
      "Properties": {
        "ManagedPolicyName": "policy-abcde-client-log-group",
        "PolicyDocument": {
          "Version": "2012-10-17",
          "Statement": [
            {
              "Effect": "Allow",
              "Action": [
                "logs:CreateLogGroup",
                "logs:CreateLogStream",
                "logs:PutLogEvents"
              ],
              "Resource": "arn:aws:logs:*:*:*"
            }
          ]
        }
      }
    }
  }
}
```

- [Localstack log output](https://github.com/localstack/localstack/files/5873071/localstack.log)

**Comment by whummer** [comment]
Thanks for reporting @Blikingor . This should be fixed in #3519 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by Blikingor** [comment]
I confirm it now works as expected. Thanks for the quick work @whummer!
